spo-links.json

```diff
@@ -92,6 +92,7 @@
 	        { "url": "http://relays.cardano.fans:9002/api/submit/tx", "pool": "CRFA", "location": "Germany" },
 		{ "url": "https://degashtz.ddns.net/api/submit/tx", "pool": "DEGAS", "location": "Germany" },
 		{ "url": "http://submit.cardano-blockhouse.de:8090/api/submit/tx", "pool": "CBH", "location": "Germany" }
+		{ "url": "https://submit.api.adahou.se/api/submit/tx", "pool": "ADAHS", "location": "Germany" }
```

Comment:
Missed adding a comma to previous entry. Also please rebase or start from latest. I will validate endpoint so we can move faster next time.